url.bs

```diff
@@ -2858,10 +2866,19 @@ method, when invoked, must run these steps:
  <li><p>Run the <a for=URLSearchParams>update steps</a>.
 </ol>
 
-<p>The
-<dfn method for=URLSearchParams><code>has(<var>name</var>)</code></dfn>
-method, when invoked, must return true if there is a name-value pair whose name is <var>name</var>
-in <a for=URLSearchParams>list</a>, and false otherwise.
+<hr>
+
+<p>The <dfn method for=URLSearchParams><code>sort()</code></dfn> method, when invoked, must run
+these steps:
+
+<ol>
+ <li><p>Lexicographically sort all name-value pairs, if any, by name, while preserving the relative
```

Comment:
The [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) docs for Array.prototype.sort explicitly calls out using "Unicode codepoint order". I'd recommend the same wording here.